rspace/src/main/scala/coop/rchain/rspace/history/package.scala

```diff
@@ -388,27 +379,27 @@ package object history {
           // we are trying to delete.  We then decide how to handle the other child based on
           // whether or not it is a Node or a Leaf
           case c @ Vector(_, _) =>
-            val (otherPointer, otherByte) = c.collect {
+            val (otherPtr, otherByte) = c.collect {
               case (child, childByte) if childByte != byte => (child, childByte)
             }.head
-            otherPointer match {
+            otherPtr match {
               // If the other child is a Node we propagate the structure as it should get collapsed
               case NodePointer(hash) =>
                 store.get(txn, hash) match {
```

Comment:
This could be removed with an additional `Pointer` type (`SkipPointer`) which would hold the affix. This is a fairly large change and would affect insertion.

Reply:
Yes, let's talk about doing that down the road.